models/configbuilder_test.go

```diff
@@ -3,13 +3,16 @@ package models
 import (
 	"testing"
 
-	"github.com/bitrise-io/bitrise-init/steps"
```

Comment:
This caused an import cycle after adding `models.RepoAccess` and depending on it from `steps`. I think this test code shouldn't depend on the `steps` package anyway, so I updated the test setup code.